airbrakes/airbrakes.py

```diff
@@ -129,31 +138,53 @@ def update(self) -> None:
         if self.est_data_packets:
             self.processed_data_packets = self.data_processor.get_processed_data_packets()
 
+        # Gets the apogee predictor packets
+        apogee_predictor_data_packets = self.apogee_predictor.get_prediction_data_packets()
+        if apogee_predictor_data_packets:
+            # We only want to update the apogee predictor data packets if we have new data
+            self.apogee_predictor_data_packets = apogee_predictor_data_packets
```

Comment:
let's assign `self.apogee..` to `None` when no packets were fetched, so we only get them for CoastState